apiserver/facades/client/client/bundles.go

```diff
@@ -14,7 +15,12 @@ import (
 // This call is deprecated, clients should use the GetChanges endpoint on the
 // Bundle facade.
 func (c *Client) GetBundleChanges(args params.BundleChangesParams) (params.BundleChangesResults, error) {
-	bundleAPI, err := bundle.NewBundle(c.api.auth)
+	model, err := c.api.state().Model()
+	if err != nil {
+		return params.BundleChangesResults{}, common.ServerError(err)
+	}
+	modelTag := model.ModelTag()
+	bundleAPI, err := bundle.NewBundleAPI(c.api.state(), c.api.auth, modelTag)
```

Comment:
It looks like the ModelTag method on state doesn't exist, but should. Feel free to add it.